app/models/ontology.rb

```diff
@@ -158,6 +158,12 @@ def all_sentences
       where('imported = ? OR imported = ?', true, false)
   end
 
+  def all_axioms
+    Axiom.unscoped.
+      where(ontology_id: self).
+      where('imported = ? OR imported = ?', true, false)
```

Comment:
what is the reason of this `where` clause, given that `imported boolean DEFAULT false NOT NULL`? Could `imported` be anything else than `true` or `false`?